atd-vze/src/views/Crashes/Crash.js

```diff
@@ -55,6 +55,16 @@ const calculateYearsLifeLost = people => {
 };
 
 function Crash(props) {
+  const [isCOA, setIsCOA] = useState(false);
+  useEffect(() => {
+    const storedEmail = window.localStorage.getItem("hasura_user_email");
```

Comment:
See https://github.com/cityofaustin/atd-vz-data/blob/61869d6f920b359a7d8aad3302c4699e973d1c2d/atd-vze/src/auth/authContext.js#L70 for where this is set.